mkdocs.yml

```diff
@@ -29,12 +29,16 @@ nav:
       - FrameIterator: api/iterators/frame-iterator.md
     - DataFrame: api/data-frame.md
     - Table: api/table.md
-  - Pixeltable Functions:
-    - Huggingface: api/functions/huggingface.md
+  - Built-in UDFs:
```

Comment:
It's a bit weird to call this "built-in user-defined functions" (they're built-in, not user-defined).

We'll also have to incorporate operators into this section, similar to how Postgres structures their docs.